accord-core/src/main/java/accord/impl/AbstractFetchCoordinator.java

```diff
@@ -231,16 +234,37 @@ void abort(Ranges abort)
         // TODO (expected): implement abort
     }
 
-    public static class FetchRequest extends WaitUntilAppliedAndReadData
+    public static class FetchRequest extends ReadData
     {
+        private static final ExecuteOn EXECUTE_ON = new ExecuteOn(Applied, Applied);
+        public final PartialTxn read;
```

Comment:
honestly I kept this `PartialTxn` because it didn't impact anything and I needed to get green before really thinking about it... We only actually care about `accord.api.Read` so would be a trivial refactor to switch to that